test_runner/regress/test_compatibility.py

```diff
@@ -438,6 +438,14 @@ def check_neon_works(
         test_output_dir / "dump-from-wal.filediff",
     )
 
+    # TODO: Run pg_amcheck unconditionally after the next release
+    try:
+        pg_bin.run(["psql", connstr, "--command", "CREATE EXTENSION amcheck"])
```

Comment:
will you add `IF NOT EXISTS`?  because you can see the exception if it was already installed.

Reply:
I think, in this particular case it doesn't affect anything, but let's add it.

Reply:
if we can avoid potential problem, let's avoid it :D